cli/config/compose/stacks/stack-monitoring/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.3'
+services:
+  monitoringEs:
+    environment:
+      - bootstrap.memory_lock=true
+      - discovery.type=single-node
+      - ES_JAVA_OPTS=-Xms512m -Xmx512m
+      - xpack.security.enabled=false
+      - xpack.monitoring.collection.enabled=true
+      - ELASTIC_USERNAME=elastic
+      - ELASTIC_PASSWORD=p4ssw0rd
+    image: "docker.elastic.co/elasticsearch/elasticsearch:${stackVersion}"
+    ports:
+      - "${monitoringEsPort:-9200}:9200"
```

Comment:
Should this file be under `cli/config/compose/services/` like the other services?

Reply:
No, because it represents the dependent services for the stack-monitoring tests. As I mentioned above, this is a stack.